bedrock/base/templates/includes/global-nav.html

```diff
@@ -1,295 +0,0 @@
-<nav role="navigation" class="moz-global-nav" id="moz-global-nav">
-    <div class="nav-horizontal-menu">
-
-      <button type="button" class="nav-button-menu nav-hidden" id="nav-button-menu" data-link-type="nav" data-link-position="open button" data-link-name="expand" data-link-group="nav element">
-        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 25" width="12" height="25">
-          <title>{{_('Menu')}}</title>
-            <rect class="p1" fill="#231f20" x="3" width="5" height="6"></rect>
-            <rect class="p2" fill="#231f20" x="3" y="9.5" width="5" height="6"></rect>
-            <rect class="p3" fill="#231f20" x="3" y="19" width="5" height="6"></rect>
-        </svg>
-      </button>
-
-      <h2 class="nav-logo">
-        <a href="{{ url('mozorg.home') }}">
-          {# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
```

Comment:
I don't see this in the new nav file. Not a blocker for removing this code but if @rraue thinks it is important from an SEO perspective we should migrate it

Reply:
Oh good call, I'll update to make sure we don't lose this 👍

Reply:
Updated to make sure we preserve this in the new navigation.